bHY6yh-d61931/cod-bqe7y7_bHY6yh-d61931.xml

```diff
@@ -0,0 +1,182 @@
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 4</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+               <name>Erin Kurian</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 4</title>
+               <date when="2023-12-21">May 18, 2021</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="P" n="cod-bqe7y7">Pal.lat.402</witness> 
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-12-21" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#P" n="128-v"/>
+            <cb ed="#P" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d61931"><!-- p3l5-r4 -->
+            <head xml:id="bHY6yh-d61931-Hd1e108">Rubrica 4</head>
+            <head xml:id="bHY6yh-d61931-Hd1e111" type="question-title">De interrogationis et responsionis prudentia</head>
+            <p xml:id="bHY6yh-d61931-d1e114">
+               <lb ed="#P"/>De enunciatione iudicatione 
+               <lb ed="#P"/>affirmatione / et reproba<lb ed="#P" break="no"/>tione 
+               capitulum v
+            </p>
```

Comment:
This scribe messed things up pretty bad. As long as these rubrics are one off, let's keeping putting `sic` around them so that we know that it is his mistake and not ours.
```suggestion
            <p xml:id="bHY6yh-d61931-d1e114">
               <lb ed="#P"/><sic>De enunciatione iudicatione 
               <lb ed="#P"/>affirmatione / et reproba<lb ed="#P" break="no"/>tione 
               capitulum v</sic>
            </p>
```